### DISCUSSION TITLE
Import doesn't work in lang dir

### DISCUSSION THREAD

**Discussion by kenan-dfrn** [nuxt-modules/i18n#2205] [Q&A]
it looks like you can't import js files into the dir directory. This in order to separate the translations into small files

en.js
- en/auth.js
- ...

**Comment by kazupon** [comment]
Nuxt i18n module provide multiple files lazy loading
I think you can work-around it.
https://v8.i18n.nuxtjs.org/guide/lazy-load-translations#multiple-files-lazy-loading
